CRITICAL - 2023-01-21 17:46:11 --> Undefined variable $grid_reunioes
in APPPATH\Views\grid_reunioes.php on line 46.
 1 APPPATH\Views\grid_reunioes.php(46): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $grid_reunioes', 'APPPATH\\Views\\grid_reunioes.php', 46)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\grid_reunioes.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('grid_reunioes', [], true)
 5 APPPATH\Controllers\Home.php(10): view('grid_reunioes')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Home->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 18:49:27 --> Call to undefined function form_open()
in APPPATH\Views\ata.php on line 23.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('ata', [], true)
 4 APPPATH\Controllers\Atas.php(38): view('ata')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Atas->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Atas))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 18:49:48 --> Call to undefined function form_open()
in APPPATH\Views\ata.php on line 23.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('ata', [], true)
 4 APPPATH\Controllers\Atas.php(38): view('ata')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Atas->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Atas))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 18:56:08 --> Call to undefined function form_open()
in APPPATH\Views\ata.php on line 23.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('ata', [], true)
 4 APPPATH\Controllers\Atas.php(38): view('ata')
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Atas->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Atas))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 19:50:59 --> syntax error, unexpected variable "$routes"
in APPPATH\Config\Routes.php on line 36.
 1 SYSTEMPATH\CodeIgniter.php(802): CodeIgniter\Router\RouteCollection->loadRoutes()
 2 SYSTEMPATH\CodeIgniter.php(449): CodeIgniter\CodeIgniter->tryToRouteIt(null)
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 20:29:42 --> Undefined variable $dadosAtas
in APPPATH\Views\topicos.php on line 34.
 1 APPPATH\Views\topicos.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $dadosAtas', 'APPPATH\\Views\\topicos.php', 34)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\topicos.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('topicos', [], true)
 5 APPPATH\Controllers\Topicos.php(19): view('topicos')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 20:35:31 --> Undefined variable $dadosAtas
in APPPATH\Views\topicos.php on line 34.
 1 APPPATH\Views\topicos.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $dadosAtas', 'APPPATH\\Views\\topicos.php', 34)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\topicos.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('topicos', [], true)
 5 APPPATH\Controllers\Topicos.php(19): view('topicos')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 20:38:30 --> Undefined variable $dadosAtas
in APPPATH\Views\topicos.php on line 34.
 1 APPPATH\Views\topicos.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $dadosAtas', 'APPPATH\\Views\\topicos.php', 34)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\topicos.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('topicos', [], true)
 5 APPPATH\Controllers\Topicos.php(19): view('topicos')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 20:41:16 --> Undefined variable $dadosAtas
in APPPATH\Views\topicos.php on line 34.
 1 APPPATH\Views\topicos.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $dadosAtas', 'APPPATH\\Views\\topicos.php', 34)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\topicos.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('topicos', [], true)
 5 APPPATH\Controllers\Topicos.php(19): view('topicos')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 20:43:47 --> Undefined variable $dados_atas
in APPPATH\Views\topicos.php on line 34.
 1 APPPATH\Views\topicos.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $dados_atas', 'APPPATH\\Views\\topicos.php', 34)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\topicos.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('topicos', [], true)
 5 APPPATH\Controllers\Topicos.php(19): view('topicos')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 20:44:24 --> Undefined variable $topicos
in APPPATH\Views\topicos.php on line 34.
 1 APPPATH\Views\topicos.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $topicos', 'APPPATH\\Views\\topicos.php', 34)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\topicos.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('topicos', [], true)
 5 APPPATH\Controllers\Topicos.php(19): view('topicos')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-01-21 20:47:05 --> Undefined variable $topicos_atas
in APPPATH\Views\topicos.php on line 34.
 1 APPPATH\Views\topicos.php(34): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $topicos_atas', 'APPPATH\\Views\\topicos.php', 34)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\topicos.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1177): CodeIgniter\View\View->render('topicos', [], true)
 5 APPPATH\Controllers\Topicos.php(19): view('topicos')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-21 20:50:15 --> mysqli_sql_exception: Unknown column 'a.cod' in 'field list' in F:\Projetos\intramet\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 F:\Projetos\intramet\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `a`.`cod...', 0)
#1 F:\Projetos\intramet\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `a`.`cod...')
#2 F:\Projetos\intramet\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `a`.`cod...')
#3 F:\Projetos\intramet\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`cod...', Array, false)
#4 F:\Projetos\intramet\app\Controllers\Topicos.php(31): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Projetos\intramet\app\Controllers\Topicos.php(20): App\Controllers\Topicos->dadosAtas()
#6 F:\Projetos\intramet\system\CodeIgniter.php(934): App\Controllers\Topicos->index()
#7 F:\Projetos\intramet\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
#8 F:\Projetos\intramet\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Projetos\intramet\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-01-21 20:50:16 --> Unknown column 'a.cod' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`cod`, `a`.`data`, `a`.`descricao`, `st`.`descricao` as `descsetor`
FROM `tbatas`
JOIN `tbsetores` `st` ON `st`.`cod` = `a`.`codsetor`
ORDER BY `a`.`cod` DESC', [], false)
 2 APPPATH\Controllers\Topicos.php(31): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Topicos.php(20): App\Controllers\Topicos->dadosAtas()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-21 20:50:52 --> mysqli_sql_exception: Unknown column 'a.cod' in 'field list' in F:\Projetos\intramet\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 F:\Projetos\intramet\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `a`.`cod...', 0)
#1 F:\Projetos\intramet\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `a`.`cod...')
#2 F:\Projetos\intramet\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `a`.`cod...')
#3 F:\Projetos\intramet\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`cod...', Array, false)
#4 F:\Projetos\intramet\app\Controllers\Topicos.php(24): CodeIgniter\Database\BaseBuilder->get()
#5 F:\Projetos\intramet\app\Controllers\Topicos.php(33): App\Controllers\Topicos->dadosAtas()
#6 F:\Projetos\intramet\system\CodeIgniter.php(934): App\Controllers\Topicos->index()
#7 F:\Projetos\intramet\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
#8 F:\Projetos\intramet\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 F:\Projetos\intramet\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-01-21 20:50:52 --> Unknown column 'a.cod' in 'field list'
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT `a`.`cod`, `a`.`data`, `a`.`descricao`, `st`.`descricao` as `descsetor`
FROM `tbatas`
JOIN `tbsetores` `st` ON `st`.`cod` = `a`.`codsetor`
ORDER BY `a`.`cod` DESC', [], false)
 2 APPPATH\Controllers\Topicos.php(24): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Controllers\Topicos.php(33): App\Controllers\Topicos->dadosAtas()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-21 21:26:44 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`intranet`.`tbtopicos`, CONSTRAINT `fk_ata` FOREIGN KEY (`codata`) REFERENCES `tbatas` (`cod`)) in F:\Projetos\intramet\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 F:\Projetos\intramet\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 F:\Projetos\intramet\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 F:\Projetos\intramet\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 F:\Projetos\intramet\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 F:\Projetos\intramet\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#5 F:\Projetos\intramet\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 F:\Projetos\intramet\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#7 F:\Projetos\intramet\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 F:\Projetos\intramet\app\Controllers\Topicos.php(98): CodeIgniter\BaseModel->save(Array)
#9 F:\Projetos\intramet\system\CodeIgniter.php(934): App\Controllers\Topicos->salvar()
#10 F:\Projetos\intramet\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
#11 F:\Projetos\intramet\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 F:\Projetos\intramet\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-01-21 21:26:44 --> Cannot add or update a child row: a foreign key constraint fails (`intranet`.`tbtopicos`, CONSTRAINT `fk_ata` FOREIGN KEY (`codata`) REFERENCES `tbatas` (`cod`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbtopicos` (`assunto`, `providencia`, `codresponsavel`, `codstatus`) VALUES (:assunto:, :providencia:, :codresponsavel:, :codstatus:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Topicos.php(98): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->salvar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-21 21:34:57 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`intranet`.`tbtopicos`, CONSTRAINT `fk_ata` FOREIGN KEY (`codata`) REFERENCES `tbatas` (`cod`)) in F:\Projetos\intramet\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 F:\Projetos\intramet\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 F:\Projetos\intramet\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 F:\Projetos\intramet\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 F:\Projetos\intramet\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 F:\Projetos\intramet\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#5 F:\Projetos\intramet\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 F:\Projetos\intramet\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#7 F:\Projetos\intramet\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 F:\Projetos\intramet\app\Controllers\Topicos.php(98): CodeIgniter\BaseModel->save(Array)
#9 F:\Projetos\intramet\system\CodeIgniter.php(934): App\Controllers\Topicos->salvar()
#10 F:\Projetos\intramet\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
#11 F:\Projetos\intramet\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 F:\Projetos\intramet\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-01-21 21:34:57 --> Cannot add or update a child row: a foreign key constraint fails (`intranet`.`tbtopicos`, CONSTRAINT `fk_ata` FOREIGN KEY (`codata`) REFERENCES `tbatas` (`cod`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbtopicos` (`assunto`, `providencia`, `codresponsavel`, `codstatus`) VALUES (:assunto:, :providencia:, :codresponsavel:, :codstatus:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Topicos.php(98): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->salvar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-21 21:37:37 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`intranet`.`tbtopicos`, CONSTRAINT `fk_ata` FOREIGN KEY (`codata`) REFERENCES `tbatas` (`cod`)) in F:\Projetos\intramet\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 F:\Projetos\intramet\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 F:\Projetos\intramet\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 F:\Projetos\intramet\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 F:\Projetos\intramet\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 F:\Projetos\intramet\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#5 F:\Projetos\intramet\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 F:\Projetos\intramet\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#7 F:\Projetos\intramet\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 F:\Projetos\intramet\app\Controllers\Topicos.php(98): CodeIgniter\BaseModel->save(Array)
#9 F:\Projetos\intramet\system\CodeIgniter.php(934): App\Controllers\Topicos->salvar()
#10 F:\Projetos\intramet\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
#11 F:\Projetos\intramet\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 F:\Projetos\intramet\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-01-21 21:37:37 --> Cannot add or update a child row: a foreign key constraint fails (`intranet`.`tbtopicos`, CONSTRAINT `fk_ata` FOREIGN KEY (`codata`) REFERENCES `tbatas` (`cod`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbtopicos` (`codata`, `assunto`, `providencia`, `codresponsavel`, `codstatus`) VALUES (:codata:, :assunto:, :providencia:, :codresponsavel:, :codstatus:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Topicos.php(98): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->salvar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-01-21 21:39:35 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`intranet`.`tbtopicos`, CONSTRAINT `fk_ata` FOREIGN KEY (`codata`) REFERENCES `tbatas` (`cod`)) in F:\Projetos\intramet\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 F:\Projetos\intramet\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `tb...', 0)
#1 F:\Projetos\intramet\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tb...')
#2 F:\Projetos\intramet\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tb...')
#3 F:\Projetos\intramet\system\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tb...', Array, false)
#4 F:\Projetos\intramet\system\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
#5 F:\Projetos\intramet\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 F:\Projetos\intramet\system\Model.php(729): CodeIgniter\BaseModel->insert(Array, false)
#7 F:\Projetos\intramet\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 F:\Projetos\intramet\app\Controllers\Topicos.php(98): CodeIgniter\BaseModel->save(Array)
#9 F:\Projetos\intramet\system\CodeIgniter.php(934): App\Controllers\Topicos->salvar()
#10 F:\Projetos\intramet\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
#11 F:\Projetos\intramet\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 F:\Projetos\intramet\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-01-21 21:39:35 --> Cannot add or update a child row: a foreign key constraint fails (`intranet`.`tbtopicos`, CONSTRAINT `fk_ata` FOREIGN KEY (`codata`) REFERENCES `tbatas` (`cod`))
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(2295): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tbtopicos` (`codata`, `assunto`, `providencia`, `codresponsavel`, `codstatus`) VALUES (:codata:, :assunto:, :providencia:, :codresponsavel:, :codstatus:)', [...], false)
 2 SYSTEMPATH\Model.php(329): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(729): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Topicos.php(98): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Topicos->salvar()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Topicos))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
